dev/com.ibm.ws.transport.http/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -77,7 +77,11 @@
             
         <!-- NO DEFAULT PROTOCOL VERSION VALUE: SPECIFICATION IS OPTIONAL -->
         <AD name="%endpoint.protocolVersion" description="%endpoint.protocolVersion.desc"
-            id="protocolVersion" required="false" type="String"/>    
+            id="protocolVersion" required="false" type="String"/> 
+            
+        <!-- NO DEFAULT NETTY ENABLEMENT VALUE: SPECIFICATION IS OPTIONAL -->
```

Comment:
Confused by this comment?

Reply:
If you look at the other examples on the class definition, I think it is meaning to say that by default it is not enabled, similar to other values such as port/protocol etc. However, no strong objection to removing it and we can revisit for a cleaner documentation later on.